__tests__/app.test.js

```diff
@@ -39,12 +39,29 @@ describe("/api/topics", () => {
 
 describe("/api", () => {
     test("GET: 200 - responds with an object containing all available endpoints", () => {
-
         return request(app)
         .get("/api")
         .expect(200)
         .then(({ body }) => {
             expect(body.endpoints).toEqual(endpoints)
         })
     })
-})
\ No newline at end of file
+})
+
+describe("/api/articles/:article_id", () => {
+    test("GET: 200 - responds with a specific article depending on its id", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article[0].article_id).toBe(1)
+            expect(response.body.article[0].title).toBe("Living in the shadow of a great man")
+            expect(response.body.article[0].topic).toBe("mitch")
+            expect(response.body.article[0].author).toBe("butter_bridge")
+            expect(response.body.article[0].body).toBe("I find this existence challenging")
+            expect(response.body.article[0].created_at).toBe('2020-07-09T20:11:00.000Z')
+            expect(response.body.article[0].votes).toBe(100)
+            expect(response.body.article[0].article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+        })
+    })
+})
```

Comment:
tests are required for error cases (not a valid id, id for article that does not exist)